spring-cloud-gcp-data-datastore/src/main/java/org/springframework/cloud/gcp/data/datastore/core/convert/TwoStepsConversions.java

```diff
@@ -137,8 +156,15 @@ else if (typeTargets.getFirstStepTarget() != null && typeTargets.getSecondStepTa
 	}
 
 	@Override
-	public Value convertOnWrite(Object proppertyVal) {
+	public Value convertOnWrite(Object proppertyVal, DatastorePersistentProperty persistentProperty) {
 		Object val = proppertyVal;
+
+		Function<Object, Value> writeConverter = this::convertOnWriteSingle;
```

Comment:
No big deal, but why not use the ` statement ? trueResult : falseResult ; ` syntax from the read function here ?